python/setup.py

```diff
@@ -14,43 +14,58 @@
 # limitations under the License.
 #
 
+import glob
 import os
 import shutil
 import sys
 from pathlib import Path
 
 from setuptools import find_packages
 
-from setuputils import clean_folder
-from setuputils import get_environment_option
-from setuputils import get_cli_option
-
 import versioneer
 from skbuild import setup
 
-install_requires = ['numba', 'cython']
 
 ##############################################################################
-# - Print of build options used by setup.py  --------------------------------
+# - Helper functions
+def get_cli_option(name):
```

Comment:
Any reason to move this functions to this file? Also, please don't remove the print of the variable *not* being set, I use it often when debugging logs

Reply:
I moved `get_cli_option` into this file for a couple of reasons:
1. With the changes in this PR all of the other functions in `setuputils` were unused, so inlining this one let me delete the module.
2. As we move towards a more modern build infrastructure for our Python packages (preferring tools like `pip build`, `pip install`, and `pip wheel` to `python setup.py *` commands) we are also going to want to store more of our build information statically in pyproject.toml to take advantage of modern tooling. Consolidating more logic into setup.py will help me evaluate what dynamic logic exists and where there are potential blockers to adopting the more static configurations preferred by modern tooling.

Reply:
I'm not sure what print message you were referring to. `get_cli_option` never printed anything in the else, it was always just returning False and I copied it verbatim. Were you perhaps referring to the prints from `get_environment_option` for `CUDA_HOME` and `CUML_BUILD_PATH`? If so, I removed those calls altogether because they were not being used. My guess is that they were used before the scikit-build conversion, but now that the Python builds use CMake to detect these things they're no longer necessary (If you left them there intentionally for logging purposes only, I'm not quite sure what benefit you get from that over the CMake logs).